res/TensorFlowPythonExamples/examples/unique/__init__.py

```diff
@@ -0,0 +1,4 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="Hole")
+op_ = tf.compat.v1.unique(in_)
```

Comment:
you now need a new line at the end of this line

Reply:
Fixed, thanks.